loop.php

```diff
@@ -164,11 +164,13 @@ class="comments-link"><?php comments_popup_link( __( 'Leave a comment', 'twentyt
 			</div>
 			<!-- .entry-meta -->
 
-			<?php if ( has_post_thumbnail() ) : // check if the post has a Post Thumbnail assigned to it. ?>
+			<?php if ( has_post_thumbnail() ) { // check if the post has a Post Thumbnail assigned to it. ?>
 				<div class="featured-image">
 					<?php the_post_thumbnail(); ?>
 				</div>
-			<?php endif; ?>
+			<?php } else { ?>
```

Comment:
You can use the else: syntax here. I find it easier to read when outputing html blocks. But isn't a big deal.